ivy.xml

```diff
@@ -55,6 +55,9 @@
     <dependency conf="shim->default" org="pentaho" name="pentaho-hadoop-shims-hdp21-package" rev="${dependency.hadoop-shims-hdp21.revision}" changing="true">
       <artifact name="pentaho-hadoop-shims-hdp21-package" type="zip"/>
     </dependency>
+    <dependency conf="shim->default" org="pentaho" name="pentaho-hadoop-shims-hdp22-package" rev="${dependency.hadoop-shims-hdp22.revision}" changing="true">
+      <artifact name="pentaho-hadoop-shims-hdp22-package" type="zip"/>

```

Comment:
See comment from pentaho-hadoop-shims PR 195, should this package replace HDP 2.1?